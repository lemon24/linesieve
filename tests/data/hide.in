--section '(\S+):$' hide two
---
0
one:
1
two:
2
three:
3
