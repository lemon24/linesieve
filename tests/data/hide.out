0
one
1
.
three
3
